stable/concourse/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: concourse
-version: 3.7.2
+version: 3.7.3
```

Comment:
Given that this ends up adding new functionality, would it make more sense to have it as a `minor` bump instead of a `patch`?

> MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards-compatible manner, and
PATCH version when you make backwards-compatible bug fixes.

Reply:
@cirocosta Regarding semantic versioning, I fully agree.
I decided to make a `patch`, because I didn't break the contract at any point and improved documentation, but of course, I have added one feature.
I will bump the version

Reply:
Fixed.